cvmfs/quota_posix.cc

```diff
@@ -630,7 +656,7 @@ pid_t PosixQuotaManager::GetPid() {
   cmd.command_type = kPid;
   cmd.return_pipe = pipe_pid[1];
   WritePipe(pipe_lru_[1], &cmd, sizeof(cmd));
-  ReadHalfPipe(pipe_pid[0], &result, sizeof(result));
+  ManagedReadHalfPipe(pipe_pid[0], &result, sizeof(result));
```

Comment:
Contrary to my previous comment: do we need to make a call to the cache manager at all anymore? Actually we already know the PID, don't we...

Reply:
That's true. I'm returning the cached pid now if available, which actually should always be the case.

Reply:
This has revealed another issue: The pid returned from the ManagedExec call may not be the final one due to the call to Daemonize here: https://github.com/cvmfs/cvmfs/blob/devel/cvmfs/quota_posix.cc#L980. Will have to rethink how to get this pid

Reply:
The easiest way will  certainly be to avoid the Daemonize, i.e running in `foreground`. Since the ManagedExec  already does a `double_fork`, this should be anyway somewhat redundant, right?